### PR TITLE
draft for gas refill script on fantom chain

### PR DESCRIPTION
**Pull request by jorijnsmit**
to solve #157 

open question for me still is whether the `ADDRESSES_FANTOM` dict is complete; do we have keepers or a botsquad wallet online yet for example? i guess we can add along the way.

treasury ops msig that sources the ftm is empty now, but with some test funding script runs well:
```
ops_executor1
snapshot result for 0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F:
              balance_before          balance_after          balance_delta
symbol                                                                    
FTM     0.000000000000000000  10.000000000000000000  10.000000000000000000 

treasury_ops_multisig
snapshot result for 0x337a32FA07eD51Aae1a7923427063B299A2307bd:
                balance_before          balance_after           balance_delta
symbol                                                                       
FTM     100.000000000000000000  90.000000000000000000  -10.000000000000000000 
```

### CODE REVIEW

**Comment by jorijnsmit** [comment]
ahh also ftm amounts are quite arbitrary, i just used eth values from mainnet * 10.

File: scripts/badger/refill_gas_ftm.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        # figure out deposit amount and transfer
+        if ftm_balance < Wei(f'{post_topup_amount} ether'):
+            # needs top-up
+            wallets[wallet_name] = GreatApeSafe(wallet_address)
+            wallets[wallet_name].take_snapshot(tokens=[])
+            top_up = Wei(f'{post_topup_amount} ether') - ftm_balance
+            if top_up < Wei(f'{MIN_TRANSFER} ether'):
+                top_up = Wei(f'{MIN_TRANSFER} ether')
+            total_ftm += Wei(top_up).to('ether')
```
Comment (jalbrekt85) [inline scripts/badger/refill_gas_ftm.py:52]: getting an error, `TypeError: Cannot convert int to decimal` coming from the `to.('ether')`. was able to be fix with something like: `total_ftm += top_up / 1e18`. also very minor but `Wei` is able to be compared with strings so L50 can just be `if top_up < = f'{MIN_TRANSFER} ether'`

**Review by jalbrekt85 — COMMENTED** [review]
removed all ftm from badger wallets to test, everything works except getting an error on L52

File: scripts/badger/refill_gas_ftm.py (hunk 1 above)
Comment (jorijnsmit) [reply scripts/badger/refill_gas_ftm.py]: i pushed fixes

**Review by jalbrekt85 — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]

**Pull request merged**